Improve the first “Typing React State” exercise

chasenlehara opened the pull request:
This:

- Refactors all the JSX to use semantic HTML
- Rewrites the comments to prompt participants to finish the exercise

Exercise:

- [Problem](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/typing-react-state-exercise-1/typing-react-state/Exercise1/problem?file=src/App.tsx)
- [Solution](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/typing-react-state-exercise-1/typing-react-state/Exercise1/solution?file=src/App.tsx)

CMLabay commented on typing-react-state/Exercise1/problem/src/App.tsx:
```diff
@@ -1,16 +1,21 @@
-import React, { useState, useEffect } from "react";
+Exercise:
+
+1) Fix the `companyDescription` state to allow a `null` value.
```
I think this may need to be: `to allow strings or null`. Possibly? I read it and was thrown off because the companyDescription already allows that. Thoughts?

chasenlehara replied:
Good call. I changed it to:

> 1) Fix the `companyDescription` state to allow a string or null value.

CMLabay reviewed (approved):
I think this looks great. Just left a comment that may or may not need addressing. If you think it's fine just merge it on in.

avinavdixit25 reviewed (approved):
Looks good!

chasenlehara committed:
Improve the first “Typing React State” exercise

This:

- Refactors all the JSX to use semantic HTML
- Rewrites the comments to prompt participants to finish the exercise
